.github/workflows/pr-labels.yml

```
@@ -0,0 +1,17 @@
+name: PR Labeler
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: WalshyDev/pr-labels@v1
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
```

We can use the token for @TheBusyBot here.

```suggestion
          token: "${{ secrets.ACCESS_TOKEN }}"
```